sources/packages/backend/apps/api/src/route-controllers/assessment/models/assessment.dto.ts

```diff
@@ -60,6 +62,8 @@ export class AssessmentNOAAPIOutDTO {
   offeringStudyEndDate: string;
   msfaaNumber: string;
   disbursement: Record<string, string | number>;
+  noaApprovalStatus: AssessmentStatus;
```

Comment:
Not part of your PR, i see, NOA and assessment share the the same AssessmentStatus enum, why we do not have a separate enum for it? If the status is reused, can we please rename or am I missing something? @dheepak-aot @andrewsignori-aot @andrepestana-aot @henryoforeh-aot

Reply:
mm... I didn't get it both are the same right? 
![image](https://user-images.githubusercontent.com/77353155/195953647-d4009349-6ff8-495e-95b0-7655f66beee0.png)

Reply:
There is nothing being shared, as shown below. At one point in time, we renamed the NOA column on DB, but the NOA column has its status defined by the enum `AssessmentStatus`.
![image](https://user-images.githubusercontent.com/61259237/195954806-8a913e70-7098-4f48-a6da-d198a4039087.png)

Reply:
okay, it looks more like the status is shared. Thanks for the explanation